php_phongo.c

```diff
@@ -3497,8 +3504,8 @@ static void* php_phongo_calloc(size_t num_members, size_t num_bytes) /* {{{ */
 	return pecalloc(num_members, num_bytes, 1);
 } /* }}} */
 
-static void* php_phongo_realloc(void* mem, size_t num_bytes)
-{ /* {{{ */
+static void* php_phongo_realloc(void* mem, size_t num_bytes) /* {{{ */
+{
```

Comment:
This is an old code formatting issue I came across while working on the vtable commits.